### PR TITLE
Updated Day 5 and Others folders.

### PR DESCRIPTION
**Pull request by prachisinhaa**
Added the question.md folder for recently posted questions.

### CODE REVIEW

File: Day-5/q2-Rotate Image/question.md
Comment (bh-g) [inline Day-5/q2-Rotate Image/question.md:1]: Hey Prachi, this ques is already added in Others/

File: Others/Best time to buy stock and sell/question.md
Comment (bh-g) [inline Others/Best time to buy stock and sell/question.md:1]: This ques is also present in the Others/

File: Day-5/q2-Rotate Image/question.md
Comment (prachisinhaa) [reply Day-5/q2-Rotate Image/question.md:1]: Oh, I didnt know, this was added by Mansi for day 5  q2

Comment (bh-g) [reply Day-5/q2-Rotate Image/question.md:1]: Yeah, but it was already there in the repo, so I had to close that issue

**Pull request closed without merging**